recipes/strling/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cp "${CC}" "${BUILD_PREFIX}/bin/gcc"
+nimble install -y --verbose
```

Comment:
I tried to use `nimble` locally just now, and I think you should run `nimble build -y --verbose` instead here (with `install`, it will also copy a symlink to `~/.nimble/bin`, which isn’t really needed).

Reply:
also right